### ISSUE TITLE
132-column mode doesn’t work correctly

### ISSUE THREAD

**Issue by ssterling** [larsbrinkhoff/terminal-simulator#54]
I’ve got no idea whether this is a ROM issue or something in the emulation, but I’m throwing it here anyway.

When running latest (0fe6abf) vt100 with no arguments, and [vttest](https://invisible-island.net/vttest/)’s “Test of cursor movements”:

![image](https://github.com/user-attachments/assets/59465bcb-466b-46d7-b13d-b46961da0aa8)

Same test, but `vt100 -Q`:

![image](https://github.com/user-attachments/assets/03550ecf-04b3-4474-836d-ff6ba9c1994b)

**Comment by larsbrinkhoff** [comment]
Correct, it currently doesn't work.  It was fine a while ago, but somewhere along the way I broke it.

**Comment by ssterling** [comment]
At the very least, I can say it doesn’t work in v0.1 (86823ec).
